StellarApp/Protocol/OrbitRotationFiles.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  OrbitRotationFiles.swift
```

Comment:
Conventionally we don't say `File` in a file name unless it describes a `File` object